multiworld/envs/pygame/point2d.py

```diff
@@ -366,6 +369,15 @@ def __init__(
                     self.inner_wall_max_dist,
                 )
             ]
+        if wall_shape == "--":
+            self.walls = [
```

Comment:
I don't think this code/variable is actually ever run at all, is this meant to be there?

Reply:
Yes, it's in __init__ and self.walls is used in the base class for drawing and collision checking